src/App.vue

```diff
@@ -1,8 +1,132 @@
 <script lang="ts" setup>
+const toggleDrawer = () => {
+  const drawer = document.querySelector('.nav-links')
+  drawer?.classList.toggle('active')
+}
+
+const reloadPage = () => {
+  location.reload()
+}
+
+const scrollToElement = (id: string) => {
+  const drawer = document.querySelector('.nav-links')
+  drawer?.classList.toggle('active')
+
+  document.getElementById(id)?.scrollIntoView()
+}
```

Comment:
Suggest not scrolling element by binding its ID. Using the real instance of the Html Element should be safer.
The core implementation has already pushed to this branch, suggest use them.

Reply:
Addressed at 1cfce42 and b56c4e9.